src/util/Logger.ts

```diff
@@ -1,24 +1,43 @@
 import chalk from 'chalk'
-
 import { Webhook } from './Webhook'
+import { Ntfy } from './Ntfy'
 
-export function log(isMobile: boolean | 'main', title: string, message: string, type: 'log' | 'warn' | 'error' = 'log', color?: keyof typeof chalk): void {
+export async function log(isMobile: boolean | 'main', title: string, message: string, type: 'log' | 'warn' | 'error' = 'log', color?: keyof typeof chalk) {
     const currentTime = new Date().toLocaleString()
     const platformText = isMobile === 'main' ? 'MAIN' : isMobile ? 'MOBILE' : 'DESKTOP'
     const chalkedPlatform = isMobile === 'main' ? chalk.bgCyan('MAIN') : isMobile ? chalk.bgBlue('MOBILE') : chalk.bgMagenta('DESKTOP')
 
-    // Clean string for the Webhook (no chalk)
+    // Clean string for Webhook and NTFY (no chalk formatting)
     const cleanStr = `[${currentTime}] [PID: ${process.pid}] [${type.toUpperCase()}] ${platformText} [${title}] ${message}`
 
-    // Send the clean string to the Webhook
-    Webhook(cleanStr)
+    // Send to Webhook if enabled
+    await Webhook(cleanStr)
+
+    // Define conditions for sending to NTFY
+    const ntfyConditions = {
+        log: [
+            message.toLowerCase().includes('started tasks for account'),
+            message.toLowerCase().includes('press the number'),
+            message.toLowerCase().includes('this script collected'),
```

Comment:
```suggestion
            message.toLowerCase().includes('the script collected'),
```
well maybe this is why